src/Tests/Microsoft.NET.Sdk.WorkloadManifestReader.Tests/SdkDirectoryWorkloadManifestProviderTests.cs

```diff
@@ -293,6 +279,82 @@ var sdkDirectoryWorkloadManifestProvider
                 .BeEquivalentTo("6.0.100", "5.0.100");
         }
 
+        [Fact]
+        public void ItShouldFallbackWhenPreviewFeatureBandHasNoManifests()
+        {
+            var testDirectory = _testAssetsManager.CreateTestDirectory().Path;
+            var fakeDotnetRootDirectory = Path.Combine(testDirectory, "dotnet");
+
+            // Write 4.0.100 manifests-> android only
+            CreateMockManifest(fakeDotnetRootDirectory, "4.0.100", "iOS");
+
+            // Write 5.0.100 manifests-> android
+            CreateMockManifest(fakeDotnetRootDirectory, "5.0.100", "Android");
+
+            // Write 6.0.100-preview.2 manifests-> ios only
+            CreateMockManifest(fakeDotnetRootDirectory, "6.0.100-preview.2", "iOS");
+
+            // Write 7.0.100 manifests-> ios and android
+            CreateMockManifest(fakeDotnetRootDirectory, "7.0.100", "Android");
+            CreateMockManifest(fakeDotnetRootDirectory, "7.0.100", "iOS");
+
+            var knownWorkloadsFilePath = Path.Combine(fakeDotnetRootDirectory, "sdk", "6.0.100-preview.4.12345", "IncludedWorkloadManifests.txt");
```

Comment:
Nit: Avoid repeating `"6.0.100-preview.4.12345"` twice, break it out into a variable or constant.